packages/api/src/promise/index.ts

```diff
@@ -36,70 +36,98 @@ import SubmittableExtrinsic from './SubmittableExtrinsic';
  * <BR>
  *
  * ```javascript
- * import ApiPromise from '@polkadot/api/promise';
+ * import Api from '@polkadot/api/promise';
  *
- * // initialise via static create
- * const api = await ApiPromise.create();
+ * async function main () {
+ *   // Initialise via static create
+ *   const api = await Api.create();
  *
- * // make a subscription to the network head
- * api.rpc.chain.subscribeNewHead((header) => {
- *   console.log(`Chain is at #${header.blockNumber}`);
- * });
+ *   // Make a subscription to the network head
+ *   await api.rpc.chain.subscribeNewHead((header) => {
+ *     console.log(`Chain is at best block #${header.blockNumber}`);
+ *   });
+ * }
+ *
+ * main().catch(console.error);
  * ```
  * <BR>
  *
  * Subscribing to chain state -
  * <BR>
  *
  * ```javascript
- * import { ApiPromise } from '@polkadot/api';
+ * import moment from 'moment';
+ *
+ * import Api from '@polkadot/api/promise';
  * import WsProvider from '@polkadot/rpc-provider/ws';
  *
- * // initialise a provider with a specific endpoint
- * const provider = new WsProvider('wss://example.com:9944')
+ * async function main () {
+ *   // Initialise a provider with a specific endpoint
+ *   const provider = new WsProvider('wss://example.com:9944');
  *
- * // initialise via isReady & new with specific provider
- * const api = await new ApiPromise(provider).isReady;
+ *   // Initialise via isReady & new with specific provider
+ *   const api = await new Api(provider).isReady;
  *
- * // retrieve the block target time
- * const blockPeriod = await api.query.timestamp.blockPeriod().toNumber();
- * let last = 0;
+ *   // Retrieve the block target time in type `Moment` and convert into seconds
+ *   const blockPeriod = await api.query.timestamp.blockPeriod();
+ *   const blockPeriodSeconds = moment(blockPeriod).unix();
+ *   let last = 0;
  *
- * // subscribe to the current block timestamp, updates automatically (callback provided)
- * api.query.timestamp.now((timestamp) => {
- *   const elapsed = last
- *     ? `, ${timestamp.toNumber() - last}s since last`
- *     : '';
+ *   // Subscribe to the current block timestamp, updates automatically (callback provided)
+ *   api.query.timestamp.now((timestamp) => {
+ *     const timestampSeconds = moment(timestamp).unix();
+ *     const elapsed = last
+ *       ? `, ${timestampSeconds - last}s since last`
+ *       : '';
  *
- *   last = timestamp.toNumber();
- *   console.log(`timestamp ${timestamp}${elapsed} (${blockPeriod}s target)`);
- * });
+ *     last = timestampSeconds;
+ *     console.log(`Timestamp ${timestampSeconds}${elapsed} (${blockPeriodSeconds}s target)`);
+ *   });
+ * }
+ *
+ * main().catch(console.error);
  * ```
  * <BR>
  *
  * Submitting a transaction -
  * <BR>
  *
  * ```javascript
- * import ApiPromise from '@polkadot/api/promise';
+ * import Api from '@polkadot/api/promise';
```

Comment:
I would just remove this and revert to the original here. IT is a bit overwhelming as an in-code comment now. (Examples are great for making these valid). So rather -

- revert to original
- import keyring (can just do the test one here, with an appropriate comment)
- leave rest as-is